pkg/executor/simple.go

```diff
@@ -2835,6 +2837,16 @@ func (e *SimpleExec) executeAdminSetBDRRole(s *ast.AdminStmt) error {
 	})
 }
 
+func (e *SimpleExec) executeAdminUnsetBDRRole(s *ast.AdminStmt) error {
+	if s.Tp != ast.AdminUnsetBDRRole {
+		return errors.New("This AdminStmt is not ADMIN UNSET BDR_ROLE")
+	}
+
+	return kv.RunInNewTxn(kv.WithInternalSourceType(context.Background(), kv.InternalTxnAdmin), e.Ctx().GetStore(), true, func(ctx context.Context, txn kv.Transaction) error {
```

Comment:
It is better to use `e.Ctx().Txn(true)` to get a transaction. Thus it will be controlled by user.